packages/beeq/src/components/toast/bq-toast.tsx

```diff
@@ -207,8 +210,12 @@ export class BqToast {
   // ===================================
 
   render() {
+    const style = {
+      ...(this.border && { '--bq-toast--border-radius': `var(--bq-radius--${this.border})` }),
+    };
```

Comment:
Please leave an empty line between the const declaration and the return statement.